pkg/api/types.go

```diff
@@ -221,6 +224,25 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDiskVolumeSource struct {
+	// iSCSI target portal
+	Portal string `json:"portal,omitempty" description:"iscsi target portal"`
+	// iSCSI target iqn
+	IQN string `json:"iqn,omitempty" description:"iscsi target iqn"`
```

Comment:
Please expand acronyms in comments, for teh sake of those of use who don't know what it means